File: 7KYU/Selective Fear of Numbers/solution.c
#include <stdbool.h>

bool am_i_afraid(const char *day, int num)
{
    return (strcmp(day, "Monday") == 0 && num == 12) ||
          (strcmp(day, "Tuesday") == 0 && num > 95) ||
          (strcmp(day, "Wednesday") == 0 && num == 34) ||
          (strcmp(day, "Thursday") == 0 && num == 0) ||
          (strcmp(day, "Friday") == 0 && num % 2 == 0) ||
          (strcmp(day, "Saturday") == 0 && num == 56) ||
          (strcmp(day, "Sunday") == 0 && (num == 666 || num == -666));
}

File: 7KYU/Selective Fear of Numbers/solution.java
public class Arithmophobia {
    static public boolean AmIAfraid(final String day, final int num) {
        return (day == "Monday" && num == 12) ||
       (day == "Tuesday" && num > 95) ||
       (day == "Wednesday" && num == 34) ||
       (day == "Thursday" && num == 0) ||
       (day == "Friday" && num % 2 == 0) ||
       (day == "Saturday" && num == 56) ||
       (day == "Sunday" && (num == 666 || num == -666));
    }
}